starter-code/src/app/services/access-control-log.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class AccessControlLogService {
+  logMessages: Array<{person: string, message: string, createdAt: Date}>;
+  newPerson: string;
+  newMessage: string;
```

Comment:
You are not using these two variables and you don't need the third one (newCreatedAt).